examples/olympic.js

```diff
@@ -17,6 +17,10 @@ function ingest_profile(hatch, uri) {
         const modified_date = new Date(Date.parse(modified_str));
         asset.set_last_modified_date(modified_date);
 
+        // Pull out the description.
+        const description = $profile('meta[name="description"]').attr('content');
+        asset.set_synopsis(description);
```

Comment:
I think those scripts will need some fixing first. Also, the previewer is missing video support. @wgreenberg: should these be handled as a separate issue?